src/fontra/core/serverutils.py

```diff
@@ -22,3 +27,68 @@ def getCodePointFromGlyphName(glyphName):
 @api
 def parseClipboard(data):
     return unstructure(clipboard.parseClipboard(data))
+
+
+# I have the feeling that I am overcomplicating things here, but even if not,
+# the following code should likely be moved to a different location.
+def skiaPathFromStaticGlyph(
+    staticGlyph: StaticGlyph, contourIndices: list
+) -> pathops.Path:
+    builder = pathops.OpBuilder()
+    for i, contour in enumerate(staticGlyph.path.asPath().contours):
```

Comment:
You should use the pen protocols and helper classes instead of rolling your own pen stuff.

The PackedPath class has a drawPoints(pen) method, that you can use. The skia-pathops path.getPen() returns a segment pen, whereas PackedPath.drawPoints() expects a point pen. Convert like this.

Use the fontTools.pens.pointPen.PointToSegmentPen and fontTools.pens.pointPen.SegmentToPointPen adapter classes to adapt one protocol to the other. Into a pathops path:

```python
path = pathops.Path()
fontraPath.drawPoints(PointToSegmentPen(path.getPen())
```

To _construct_ a PackedPath you should use the PackedPathPointPen.